src/plugins/inputs/setpass.js

```diff
@@ -0,0 +1,41 @@
+var bcrypt = require("bcrypt-nodejs");
+var Msg = require("../../models/msg");
+
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "setpass") {
```

Comment:
some people like the ability to not have to click ui elements to perform actions. That's why IRC has historically has slash-commands. there is no requirement for a command akin to `/command some arguments` to be forwarded to the IRC server as `COMMAND :some arguments`! *many* IRC clients use slash-commands for client-specific functionality, and I added this command based on that premise.